scripts/update.sh

```diff
@@ -10,9 +10,9 @@ GLUONDIR="$(pwd)"
 for module in $GLUON_MODULES; do
 	echo "--- Updating module '$module' ---"
 	var=$(echo "$module" | tr '[:lower:]/' '[:upper:]_')
-	eval repo=\${${var}_REPO}
-	eval branch=\${${var}_BRANCH}
-	eval commit=\${${var}_COMMIT}
+	eval repo=\$\{"${var}"_REPO\}
+	eval branch=\$\{"${var}"_BRANCH\}
+	eval commit=\$\{"${var}"_COMMIT\}
```

Comment:
How about
```bash
eval "repo=\${${var}_REPO}"
```
or
```bash
eval 'repo=${'"${var}"'_REPO}'
```
etc.? I'm not sure which I like better, the second variant trades an ugly backslash for a few additional quotes...

Reply:
I prefer last one. Applied